src/components/VideoCard/VideoCard.component.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useVideoContext } from '../../store/VideoContext';
+import {
+  VideoCardWrapper,
+  LikeButton,
+  VideoCardTitle,
+  VideoCardHeader,
+} from './VideoCardStyles';
+import likeIcon from '../../assets/like.svg';
+
+function VideoCard({ videoDescription, videoTitle, videoThumbnail, videoId }) {
+  const { dispatch } = useVideoContext();
+
+  const handleVideoClick = () => {
```

Comment:
would be better if these functions lived inside `useVideoContext` and took the payload as arguments so you don't need to worry about dispatching from here and you have the wrapper ready if you want to use these functions elsewhere